runners/flink/1.8/src/main/java/org/apache/beam/runners/flink/translation/types/CoderTypeSerializer.java

```diff
@@ -82,10 +81,14 @@ public T createInstance() {
 
   @Override
   public T copy(T t) {
```

Comment:
Maybe rename to `copyIfNeeded`, as when the flag is on, it doesn't copy anything.

Reply:
This is a Flink interface method which we can't rename.

Reply:
Ah, I missed the `@Override` annotation. Understood.